test/functional/checkupdate/chk-update-format-bump.bats

```diff
@@ -29,4 +29,16 @@ test_setup() {
 	)
 	assert_is_output "$expected_output"
 
+	# Also check if verbose options holds
```

Comment:
please add this as a new test, tests are ideally atomic unless that cannot happen for some reason. As it is now, if the previous test fails you will never know if this test would pass or fail.

Reply:
Done